### ISSUE TITLE
Option to align with previous line [SplitTooLongLine]

### ISSUE THREAD

**Issue by ButterBetzi** [MarketSquare/robotframework-tidy#484]
Lets use the example of the docs:
```*** Keywords ***
Keyword
    Keyword With Longer Name    ${arg1}    ${arg2}    ${arg3}  # let's assume that arg2 is at 120 char
```
This results in the following with "split_on_every_arg=False":
```*** Keywords ***
Keyword
    # let's assume that arg2 is at 120 char
    Keyword With Longer Name    ${arg1}
    ...    ${arg2}    ${arg3}
```
I would prefer another option which then results in the following:
```*** Keywords ***
Keyword
    # let's assume that arg2 is at 120 char
    Keyword With Longer Name    ${arg1}
    ...                         ${arg2}    ${arg3}
```

**Comment by ButterBetzi** [comment]
I tried this, yes. But like you're saying, I dont want this, this makes my testcases pretty messy.
So yes, I only want to align multiline lines to each other.

**Comment by ButterBetzi** [comment]
I now have a second case where I would like to have this functionality. I let my testcases automatically lint with robocop and automatically format with robotidy. Problem:
robotidy indendts my tags section, but it does not align with the previous line.
That is then marked by robocop as an error "Each next continuation line should be aligned with the previous one robocop[1015] (https://robocop.readthedocs.io/en/2.8.1/rules.html#misaligned-continuation-row)"

So I either have to disable some settings in robocop or in robotidy, but I dont want to because they are useful in other places :D

Example:
before:
```
*** Test Cases ***
Dummy Testcase
    [Tags]    Tag1    Tag10    Tag11    Tag12    Tag13    Tag2    Tag3    Tag4    Tag5    Tag6    Tag7    Tag8    Tag9    Tag14    Tag15
    Fail
```

after:
```
*** Test Cases ***
Dummy Testcase
    [Tags]    Tag1    Tag10    Tag11    Tag12    Tag13    Tag14    Tag15    Tag2    Tag3    Tag4    Tag5    Tag6
    ...    Tag7    Tag8    Tag9
    Fail
```

what I need:
```
*** Test Cases ***
Dummy Testcase
    [Tags]    Tag1    Tag10    Tag11    Tag12    Tag13    Tag14    Tag15    Tag2    Tag3    Tag4    Tag5    Tag6
    ...       Tag7    Tag8    Tag9
    Fail
```

**Comment by bhirsz** [comment]
It looks like something I can add - as part of the SplitTooLongLine transformer. It could be extra flag like   ``align_new_line``. Thanks for the code examples, it will be helpful im the development.

**Comment by bhirsz** [comment]
.... It was suprisingly easy to implement. However adding the documentation etc will take me some time - but it will be part of the major release planned for the end of this week. If you want, you can test the feature using this branch https://github.com/MarketSquare/robotframework-tidy/compare/SplitTooLongLine_align_new_line?expand=1 . Note that the parameter name ("align_new_line") is subject to change.

**Comment by bhirsz** [comment]
Though I have one edge case. What about this:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line    ${arg}
```

Should it be:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line
    ...                                                                       ${arg}
```

Or:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line
    ...    ${arg}
```

Probably the secon - but I wanted to confirm (currently the code uses first approach and I need to fix it).

**Comment by bhirsz** [comment]
Though I have one edge case. What about this:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line    ${arg}
```

Should it be:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line
    ...                                                                       ${arg}
```

Or:

```
    This Keyword Goes Beyond Limit And Next Argument Should Be In New Line
    ...    ${arg}
```

Probably the second - but I wanted to confirm (currently the code uses first approach and I need to fix it).

**Comment by mnojek** [comment]
IMO Second one looks way better.

**Comment by ButterBetzi** [comment]
Thank you very much!

**Comment by bhirsz** [comment]
This change was released together with Robotidy 4.0.  See the release notes on how to use it. However I recommend using Robotidy 4.0.1 when it is (soon) released since I also introduced regression.
